docs/source.rst

```diff
@@ -46,6 +36,17 @@ While using the Tor Browser on your personal computer helps hide your activity o
 .. _`Tails`: https://tails.boum.org/
 
 
+Choose who to submit to
+-----------------------
+
+Each SecureDrop instance is totally independent, and submissions to that instance are only available to journalists associated with that organization.
+
+All organizations have a *landing page* that provides their own organization-specific recommendations for using SecureDrop. We encourage you to consider an organization's landing page before submitting to them.
+
+Most organizations make their landing page prominently accessible from their main website's homepage. You can also find an incomplete list of organizations accepting submissions through SecureDrop on the `SecureDrop Directory`_ maintained by Freedom of the Press Foundation.
```

Comment:
I believe we try to actively discourage potential sources from going to our directory unless they are using Tor.

Reply:
Yes, we need to push sources toward Tor as early as possible. The new securedrop.org website will include "you should be using tor!" messages when visited over clearnet. We can also start encouraging links to the directory as Onion URLs, which has implications for #1608 and #2200. Still, that shouldn't block merge here, because the recommendations currently in this PR are indeed the best we have to date.